requirements-app.txt

```diff
@@ -23,13 +23,15 @@ SQLAlchemy==1.3.22
 strict-rfc3339==0.7
 rfc3987==1.3.8
 cachetools==4.2.0
+beautifulsoup4==4.9.3
+lxml==4.6.2
```

Comment:
we'll need to be super careful to make sure this is kept up-to-date as it's parsing untrusted xml from the public internet

Reply:
Yeah. It’s a shame there isn’t a library in [the `defusedxml` module](https://pypi.org/project/defusedxml/) which can validate XML against a schema.